is1_31.loboc

```diff
@@ -0,0 +1,18 @@
+// ISEQ1: Test logical operations
+//tlc type: run
+//tlc case:  t=t,t,t,t  
+//tlc case:  f=f,f,f,f  
+//tlc case:  f=t,f,f,f  
+//tlc case:  f=f,t,t,f 
+//tlc case:  f=f,t,f,f 
+//tlc case:  f=t,f,t,f 
+//tlc case:  f=f,f,t,f 
+//tlc case:  t=t,t,f,f
+//tlc case:  t=t,t,f,f
+//tlc case:  f=t,f,t,f
+//tlc case:  t=t,t,t,f
+//tlc case:  t=f,f,t,t
+
+bool a,b,c,d;
+(a && b) || (c && d);
+
```

Comment:
This is very similar to `is1_30.loboc`, combine?

Reply:
Done. I changed the test.